test/navigation/NavigatorSpec.scala

```diff
@@ -602,6 +602,13 @@ class NavigatorSpec extends SpecBase with TableDrivenPropertyChecks with Generat
         navigator.nextPage(UnknownPage, CheckMode, UserAnswers("id")) mustBe routes.CheckYourAnswersController.onPageLoad
       }
 
+      "must go from ReporterTypePage" - {
+        "to Check Your Answers if Individual is unchanged" in {
```

Comment:
How do you know it's unchanged? And not that it was something else, but now it's been altered to Individual?

Reply:
Good point - i've made a change where we can tell if we've remained on the individual using the existence of the NInumber yes/no page.